public/app/plugins/datasource/cloudwatch/components/LogsQueryField.tsx

```diff
@@ -1,6 +1,5 @@
-import { css, cx } from '@emotion/css';
 import { LanguageMap, languages as prismLanguages } from 'prismjs';
-import React, { ReactNode } from 'react';
+import * as React from 'react';
```

Comment:
Think it's better to import the named exports directly rather than importing all.

Reply:
Thanks for the recommendation.

I do see there are a number of ways to import. Is this part of a convention we follow at Grafana that is defined somewhere?
I did run across this answer,  https://stackoverflow.com/a/56833596 -- I guess we are using `allowSyntheticDefaultImports` here?

Reply:
To add to what Sarah said around optimisation, taking only the methods you need could potentially help with performance and memory management, depending on the kind of bundler you use. Although I'm not familiar enough with how webpack works in our project to know if it makes a significant difference and if it impacts our tree shaking.

Not really specific to this particular React import but generally, another thing it could help with is mocking what you need for tests too, with it being clear what exactly is needed to be mocked or stubbed.